src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/mapper/InductionMigrationMapperTest.kt

```diff
@@ -0,0 +1,375 @@
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration.jpa.entity.HopingToWork as HopingToWorkEntity
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration.jpa.entity.InPrisonTrainingType as InPrisonTrainingTypeEntity
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration.jpa.entity.InPrisonWorkType as InPrisonWorkTypeEntity
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration.jpa.entity.TrainingType as TrainingTypeEntity
+
+class InductionMigrationMapperTest {
+
+  private val localDateTimeMapper = LocalDateTimeMapper()
+  private val workOnReleaseMapper = WorkOnReleaseMigrationMapperImpl().also {
+    WorkOnReleaseMigrationMapperImpl::class.java.getDeclaredField("localDateTimeMapper").apply {
+      isAccessible = true
+      set(it, localDateTimeMapper)
+    }
+  }
+  private val qualificationsMapper = PreviousQualificationsMigrationMapperImpl().also {
+    PreviousQualificationsMigrationMapperImpl::class.java.getDeclaredField("localDateTimeMapper").apply {
+      isAccessible = true
+      set(it, localDateTimeMapper)
+    }
+  }
+  private val trainingMapper = PreviousTrainingMigrationMapperImpl().also {
+    PreviousTrainingMigrationMapperImpl::class.java.getDeclaredField("localDateTimeMapper").apply {
+      isAccessible = true
+      set(it, localDateTimeMapper)
+    }
+  }
+  private val workExperiencesMapper = PreviousWorkExperiencesMigrationMapperImpl().also {
+    PreviousWorkExperiencesMigrationMapperImpl::class.java.getDeclaredField("localDateTimeMapper").apply {
+      isAccessible = true
+      set(it, localDateTimeMapper)
+    }
+  }
+  private val inPrisonInterestsMapper = InPrisonInterestsMigrationMapperImpl().also {
+    InPrisonInterestsMigrationMapperImpl::class.java.getDeclaredField("localDateTimeMapper").apply {
+      isAccessible = true
+      set(it, localDateTimeMapper)
+    }
+  }
+  private val skillsAndInterestsMapper = SkillsAndInterestsMigrationMapperImpl().also {
+    SkillsAndInterestsMigrationMapperImpl::class.java.getDeclaredField("localDateTimeMapper").apply {
+      isAccessible = true
+      set(it, localDateTimeMapper)
+    }
+  }
+  private val workInterestsMapper = FutureWorkInterestsMigrationMapperImpl().also {
+    FutureWorkInterestsMigrationMapperImpl::class.java.getDeclaredField("localDateTimeMapper").apply {
+      isAccessible = true
+      set(it, localDateTimeMapper)
+    }
+  }
+
+  private val inductionMapper: InductionMigrationMapper = InductionMigrationMapper(
+    workOnReleaseMapper = workOnReleaseMapper,
+    qualificationsMapper = qualificationsMapper,
+    trainingMapper = trainingMapper,
+    workExperiencesMapper = workExperiencesMapper,
+    inPrisonInterestsMapper = inPrisonInterestsMapper,
+    skillsAndInterestsMapper = skillsAndInterestsMapper,
+    workInterestsMapper = workInterestsMapper,
+  )
+
+  @Test
+  fun `should map to InductionMigrationEntity`() {
+    // Given
```

Comment:
This is the largest `Given` section I've ever written and I can't say I'm proud of it! 😂 

I could make the builders use the same values, which would massively reduce the amount of code, but it wouldn't be as explicit 🤔